BrowserKit/Sources/WebEngine/WKWebview/WKBackForwardListItem+EngineSessionBackForwardListItem.swift

```diff
@@ -0,0 +1,6 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+import WebKit
+
+extension WKBackForwardListItem: EngineSessionBackForwardListItem {}
```

Comment:
I think we could have this extension under the file `EngineSessionBackForwardListItem.swift` since it goes hand in hand